Project.toml

```
@@ -50,6 +51,7 @@ FastPow = "0.1"
 FileIO = "1"
 ForwardDiff = "1"
 GPUArraysCore = "0.2"
+IntervalSets = "=0.7.11, >=0.8"
```

Who says it will work with 0.8? I would prefer the same as in Trixi.jl:

```suggestion
IntervalSets = "0.7 - 0.7.11"
```
Once 0.8 is out, we will get a CompatHelper PR anyway.